docker/i586-unknown-linux-gnu/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM ubuntu:17.04
```

Comment:
This should be as low as possible, i.e. 12.04 as the i686 target, to make the binaries produced with Cross compatible with legacy distributions like centos.

Reply:
Fixed. This should make the version in the readme file correct right?